guides/common/assembly_provisioning-virtual-machines-kubevirt.adoc

```diff
@@ -0,0 +1,62 @@
+
+ifeval::["{build}" == "satellite"]
+Note that template provisioning is not supported for this release.
+
+[IMPORTANT]
+====
+The {KubeVirt} compute resource is a Technology Preview feature only. Technology Preview features are
+not supported with Red Hat production service level agreements (SLAs) and might not be
+functionally complete. Red Hat does not recommend using them in production.
+These features provide early access to upcoming product features, enabling
+customers to test functionality and provide feedback during the development process.
+For more information about the support scope of Red Hat Technology Preview features,
+see https://access.redhat.com/support/offerings/techpreview/.
+====
+endif::[]
+
+.Prerequisites
+
+include::modules/snip_common-compute-resource-prereqs.adoc[]
+* A {KubeVirt} user that has the `cluster-admin` permissions for the Openshift Container Platform virtual cluster. For more information, see https://access.redhat.com/documentation/en-us/openshift_container_platform/4.1/html/authentication/using-rbac[Using RBAC to Define and Apply Permissions] in the _Authentication_ guide of the Openshift Container Platform documentation.
+* A {SmartProxyServer} managing a network on the {KubeVirt} server. Ensure that no other DHCP services run on this network to avoid conflicts with {SmartProxyServer}. For more information about network service configuration for {SmartProxyServer}s, see xref:Configuring_Networking[].
+* A {Project} user account with the following roles:
+** *Edit hosts*
+** *View hosts*
++
+For more information, see link:{BaseURL}administering_red_hat_satellite/chap-red_hat_satellite-administering_red_hat_satellite-users_and_roles#sect-{Project_Link}-Administering_{Project_Link}-Creating_and_Managing_Users-Assigning_Roles_to_a_User[Assigning Roles to a User] in the _Administering {ProjectName}_ guide.
+* A custom role in {Project} with the following permissions:
+** *view_compute_resources*
+** *destroy_compute_resources_vms*
+** *power_compute_resources_vms*
+** *create_compute_resources_vms*
+** *view_compute_resources_vms*
+** *view_locations*
+** *view_subnets*
++
+For more information about creating roles, see link:{BaseURL}administering_red_hat_satellite/chap-red_hat_satellite-Administering_{Project_Link}-Users_and_Roles#sect-red_hat_satellite-Administering_{Project_Link}-Creating_and_Managing_Roles-Creating_a_Role[Creating a Role] in the _Administering {ProjectName}_ guide. For more information about adding permissions to a role, see link:{BaseURL}administering_red_hat_satellite/chap-red_hat_satellite-Administering_{Project_Link}-Users_and_Roles#sect-red_hat_satellite-Administering_{Project_Link}-Creating_and_Managing_Roles-Adding_Permissions_to_a_Role[Adding Permissions to a Role] in the _Administering {ProjectName}_ guide.
+
+////
+.Procedure Overview
+
+. xref:adding-kubevirt-connection_kubevirt-provisioning[].
+////
```

Comment:
Why are these lines hidden?

Reply:
Because it seems a bit useless to include it at the moment, with only one step, but hopefully someone can fill out the chapter and this overview later.

Reply:
Got this.